src/PointwiseFunctions/AnalyticData/GrMhd/CMakeLists.txt

```diff
@@ -93,4 +93,37 @@ if (TARGET FUKA::Exporter)
     )
 endif()
 
+if (TARGET COCAL::Exporter)
+add_custom_command(
+  OUTPUT ${CMAKE_SOURCE_DIR}/src/PointwiseFunctions/AnalyticData/GrMhd/COCAL_reader/libcocal.a
+  COMMAND make -C ${CMAKE_SOURCE_DIR}/src/PointwiseFunctions/AnalyticData/GrMhd/COCAL_reader clean
+  COMMAND make -C ${CMAKE_SOURCE_DIR}/src/PointwiseFunctions/AnalyticData/GrMhd/COCAL_reader
```

Comment:
so why require me to point to a COCAL library above? Does it take the source code? That is somewhat strange, usually one would point to the compiled version.